docs/docs/code/nodejs/delay/README.md

```diff
@@ -94,3 +110,29 @@ export default defineComponent({
 You cannot run `$.respond` after running `$.flow.delay`. Pipedream ends the original execution of the workflow when `$.flow.delay` is called and issues the following response to the client to indicate this state:
 
 > $.respond() not called for this invocation
+
+If you need to set a delay on an HTTP request triggered workflow, consider using using [`setTimeout`](#settimeout) instead.
+
+## `setTimeout`
+
+Alternatively, you can use `setTimeout` instead of using `$.flow.delay` to delay individual workflow steps.
+
+However, there are some drawbacks to using `setTimeout` instead of `$.flow.delay`. `setTimeout` will count towards your workflows compute time, for example:
```

Comment:
```suggestion
However, there are some drawbacks to using `setTimeout` instead of `$.flow.delay`. `setTimeout` will count towards your workflow's compute time, for example:
```

Reply:
Changed "workflows" to "workflow's" to follow Dreyer's English rule on possessives.